### PR TITLE
address_query replaced in configs

### PR DESCRIPTION
**Pull request by meschw04**
# Description

Made to satisfy issue #1754 on hard-coding address queries in configs.

## Type of change

Choose which options apply, and delete the ones which do not apply.

- Bug fix (non-breaking change that fixes an issue)

### CODE REVIEW

**Comment by yadudoc** [comment]
Hey @meschw04, there are a few more configs that explicitly use `address_by_query`/`address_by_hostname`. We could either merge this PR as is, or if you need more time to finish up the rest of the files  we could do that, or I could do the remainder. Let me know what you think.

File: parsl/configs/Azure.py [hunk 1]
```diff
@@ -26,7 +26,6 @@
     executors=[
         HighThroughputExecutor(
             label='azure_single_node',
-            address=address_by_query(),
```
Comment (ZhuozhaoLi) [inline parsl/configs/Azure.py:29]: Should remove the import `address_by_query` above.

Comment (yadudoc) [reply parsl/configs/Azure.py:29]: ZZ, Azure uses address_by_query further down the config for the `storage_access` option for `RSyncStaging`. So we shouldn't remove the import.

Comment (ZhuozhaoLi) [reply parsl/configs/Azure.py:29]: Oh, I missed that.

**Pull request closed without merging**